homeassistant/components/dynalite/config_flow.py

```diff
@@ -18,19 +21,68 @@ def __init__(self) -> None:
         """Initialize the Dynalite flow."""
         self.host = None
 
+    def existing_host_entry(self, config) -> Any:
+        """Check if a specific host is already configured and return its entry or None."""
+        for entry in self.hass.config_entries.async_entries(DOMAIN):
```

Comment:
We can use `self._async_current_entries`.

Reply:
done